docker/Dockerfile

```diff
@@ -39,10 +39,9 @@ RUN apk add --no-cache \
   vim less \
   tar xz p7zip \
   python3 py3-pip \
-  && git config --global --add safe.directory /app \
-  && ln -s $(ls -1r /usr/lib/libproj.so.* | head -1) /usr/lib/libproj.so.25
+  && git config --global --add safe.directory /app
```

Comment:
Did you test any postgis functionality?

Reply:
No, but we definitely should.  Thanks for the reminder.